js/tests.js

```diff
@@ -3,3 +3,10 @@ QUnit.test("hello test", function(assert) {
 });
 
 // ADD TESTS HERE
+QUnit.test("side kick test", function(assert){
+	assert.strictEqual(plane.sideKick, "Dead", "Planes side kick was dynamically changed to dead.")
+	});
+
+QUnit.test("costume test", function(assert){
+	assert.strictEqual(car.costume, "cape", "Checks that costume is cape.")
+	});
```

Comment:
Having the instance creation and method calls within the tests will help show what each are testing, and making sure that one test is less likely to affect the other.